doc/api/cli.md

```diff
@@ -90,6 +90,13 @@ added: v8.5.0
 
 Enable experimental ES module support and caching modules.
 
+### `--experimental-policy`
+<!-- YAML
+added: TODO
+-->
+
+Use the specified file as a security policy.
```

Comment:
Are we printing a warning on the console when this is used? I think we should.

Reply:
Just because it is experimental, or more permanent? I would avoid a more permanent warning.

Reply:
Just because it’s experimental.

Reply:
fixed